modules/core/src/main/java/org/mycontroller/standalone/message/McMessage.java

```diff
@@ -54,6 +54,7 @@
     @NonNull
     private Integer gatewayId;
     private String nodeEui;
+    private String name;
```

Comment:
@Fraid I have added code for this on PR: https://github.com/Fraid/mycontroller/pull/1
Kindly check it and do merge if everything goes well